Rubberduck.Refactorings/Rename/RenameRefactoring.cs

```diff
@@ -483,19 +476,23 @@ private void RenameModule(IRewriteSession rewriteSession)
                     }
                 default:
                     {
-                        if (_vbe.Kind == VBEKind.Hosted)
+                        using (var vbe = component.VBE)
                         {
-                            // VBA - rename code module
-                            using (var codeModule = component.CodeModule)
+                            if (vbe.Kind == VBEKind.Hosted)
                             {
-                                Debug.Assert(!codeModule.IsWrappingNullReference, "input validation fail: Attempting to rename an ICodeModule wrapping a null reference");
-                                codeModule.Name = _model.NewName;
+                                // VBA - rename code module
+                                using (var codeModule = component.CodeModule)
+                                {
+                                    Debug.Assert(!codeModule.IsWrappingNullReference,
```

Comment:
This should probably log rather than assert.

Reply:
Although I agree, I will not change this in this PR; I will make a note for my next/new one.